src/02 - Object types.md

```diff
@@ -170,8 +170,8 @@ The following notation is used in the tables below:
 | M500 | string | skjerming              |          |             |     x     |  |  x   |  x   |   x   |  x   |               | <p>&bull; code list in Documaster</p>                                                                                |                                                                                   |
 |      |        |                        |          |             |           |  |      |      |       |      |               |                                                                                                                      |                                                                                   |
-|      | ref    | refArkivdel            |    c     |             |           |  |      |      |   x   |      |       x       |                                                                                                                      | <p>&bull; exactly one of **refArkivdel** and **refForelderMappe** must be set</p> |
-|      | ref    | refForelderMappe       |    c     |             |           |  |      |      |   x   |      |       x       | <p>&bull; references a parent **Mappe**</p>                                                                          | <p>&bull; exactly one of **refArkivdel** and **refForelderMappe** must be set</p> |
+|      | ref    | refArkivdel            |    c     |             |           |  |      |  x   |   x   |      |       x       |                                                                                                                      | <p>&bull; exactly one of **refArkivdel** and **refForelderMappe** must be set</p> |
+|      | ref    | refForelderMappe       |    c     |             |           |  |      |  x   |   x   |      |       x       | <p>&bull; references a parent **Mappe**</p>                                                                          | <p>&bull; exactly one of **refArkivdel** and **refForelderMappe** must be set</p> |
 |      | ref    | refBarnMappe           |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; references children **Mappe**</p>                                                                          |                                                                                   |
 |      | ref    | refRegistrering        |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; references **Basisregistrering** or **Journalpost**</p><p>&bull; may be partially supported by systems</p> | <p>&bull; references to **Journalpost** not currently supported in Documaster</p> |
 |      | ref    | refKlasse              |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; references a **Klasse** from the primary **Klassifikasjonssystem** of the **Arkivdel**</p>                 |                                                                                   |
```

Comment:
Shouldn't the `refKlasse` reference in Folder be viewable too?

Reply:
Yes, you are correct. Fixed.